src/xclim/indicators/atmos/_precip.py

```diff
@@ -815,3 +816,15 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.water_cycle_intensity,
 )
+
+antecedent_precipitation_index = Indicator(
+    title="Antecedent Precipitation Index",
+    identifier="antecedent_precipitation_index",
+    realm="atmos",
+    units="mm",
+    long_name="Antecedent Precipitation Index",
+    description="The Antecedent Precipitation Index, defined as running weighted sum of daily precipitation totals.",
```

Comment:
```suggestion
    description="Weighted moving sum of daily precipitation totals with a {window}-day window. Weights are an exponential decay of base {weights}.",
```
This is usually how we pass this kind of information. (It will also appear in the `history` of course).

Also, this is more a personal opinion, but I would write the `description` as describing the output data, instead of the computation itself. The `description` and `long_name` are added as attributes, while `title`  and `abstract` are only internal to xclim, for documentation.

Reply:
Thank you for the explanation!